src/tests/VisualStudioProjectLoaderTests.cs

```diff
@@ -326,6 +326,22 @@ public void FromProjectWithTemplatedPaths()
                     "Invalid FixedPath assembly path");
             }
         }
+
+        [Test]
+        public void ProjectWithDuplicatedSections()
+        {
+            using (var file = new TestResource("test-duplicated-key-project.csproj", NormalizePath(@"csharp-sample\csharp-sample.csproj")))
+            {
+                IProject project = _loader.LoadFrom(file.Path);
+                Assert.AreEqual(2, project.ConfigNames.Count);
+
+                var debugPackage = project.GetTestPackage("Debug");
+                Assert.AreEqual(1, debugPackage.SubPackages.Count, "Debug should have 1 assembly");
+
+                var releasePackage = project.GetTestPackage("Release");
+                Assert.AreEqual(1, releasePackage.SubPackages.Count, "Release should have 1 assembly");
+            }
+        }

```

Comment:
This test verifies that there is no problem with having a config described with multiple property groups. As such, it would be an improvement. However, it doesn't deal with the OutputPath appearing multiple times, which seems to be a part of the problem. As an improvement, I'd be glad to see it merged, but I think there is another issue needed in that case.